src/Symfony/Component/Intl/Resources/data/currencies/en_150.json

```diff
@@ -0,0 +1,10 @@
+{
+    "Version": "2.1.19.76",
+    "Names": {
+        "EUR": [
+            "€",
+            "Euro",
+            {}
```

Comment:
I know this is generated, but this means our APIs exposing currency name will return invalid data (their contract is to return an array of strings).

If there is a bug in the original files, this should be reported to the ICU project, and we should filter out invalid values in our generator script